src/Workspaces/Core/Portable/OrganizeImports/OrganizeImportsOptionsProviders.cs

```diff
@@ -2,30 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Runtime.Serialization;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.AddImport;
 using Microsoft.CodeAnalysis.Editing;
 using Microsoft.CodeAnalysis.Formatting;
 using Microsoft.CodeAnalysis.Options;
 
 namespace Microsoft.CodeAnalysis.OrganizeImports;
 
-[DataContract]
-internal readonly record struct OrganizeImportsOptions
```

Comment:
options type moved to tis own file in the CodeStyle layer.  The 'Providers' types move to the CodeFixes layer.  This removes the need for CODE_STYLE checks, and properly layers things.